test/e2e/play_kube_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var unknownKindYaml = `
-apiVerson: v1
+apiVersion: v1
```

Comment:
Changes like this worry me.

Personally, I'd rather people use things like `_` to indicate intentionally wrong things.